[electron] Error: Cannot find module '@expo/webpack-config/addons'

## Description 

Super excited about Expo with Electron! It looks like `yarn expo-electron` isn't working for me. I tried it in both an existing project and in a new one.

## Expected Behavior

It should run an electron project.

```es6
expo init expo-desktop
cd expo-desktop
yarn add -D @expo/electron-adapter
yarn expo-electron // error here
```

## Observed Behavior

It seems like `yarn expo-electron` gets held up at `@expo/webpack-config`. I checked my `package.json` after running it and I don't see the webpack config added.

```es6
$ yarn expo-electron

yarn expo-electron v0.27.5
$ "/Users/path/to/expo-electron/node_modules/.bin/expo-electron"
module.js:550
    throw err;
    ^

Error: Cannot find module '@expo/webpack-config/addons'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Module.require (module.js:597:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/path/to/expo-electron/node_modules/@expo/electron-adapter/build/Webpack.js:21:18)
    at Module._compile (module.js:653:30)
    at Object.Module._extensions..js (module.js:664:10)
    at Module.load (module.js:566:32)
    at tryModuleLoad (module.js:506:12)
    at Function.Module._load (module.js:498:3)
error Command failed with exit code 1.
```

## Environment

```
Expo CLI 3.9.1 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 8.12.0 - /usr/local/bin/node
      Yarn: yarn install v0.27.5
[1/4] Resolving packages...
success Already up-to-date.
Done in 0.72s. - /usr/local/bin/yarn
      npm: 6.13.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 11.2/11B52 - /usr/bin/xcodebuild
    npmPackages:
      @types/react: ^16.8.23 => 16.9.13 
      @types/react-native: ^0.57.65 => 0.57.65 
      expo: ^35.0.0 => 35.0.1 
      react: 16.8.3 => 16.8.3 
      react-native: https://github.com/expo/react-native/archive/sdk-35.0.0.tar.gz => 0.59.8 
    npmGlobalPackages:
      expo-cli: 3.9.1
```

## Reproducible Demo -  [Link](https://github.com/nandorojo/expo-electron-issue)


(I'm not sure if this would be useful since it's just an empty project.)

## I also tried this:

```es6
npm i electron @expo/webpack-config react-native-web 
```

Created **`electron-webpack.js`**:
```es6
const { withExpoAdapter } = require("@expo/electron-adapter");

module.exports = withExpoAdapter({
  projectRoot: __dirname
  // Provide any overrides for electron-webpack: https://github.com/electron-userland/electron-webpack/blob/master/docs/en/configuration.md
});
```

```es 6
yarn expo-electron start
```

But no luck. Thanks for your help!

--- a/packages/electron-adapter/bin/expo-electron.js
+++ b/packages/electron-adapter/bin/expo-electron.js
@@ -17,7 +17,7 @@ process.on('unhandledRejection', err => {
 });
 const { realpathSync } = require('fs-extra');
 
-const { ensureMinProjectSetupAsync } = require('../');
+const { ensureMinProjectSetupAsync } = require('../build/Config');
 
 const spawnAsync = require('@expo/spawn-async');
 

